Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/InkToolbar/InkToolbar.cs

```diff
@@ -0,0 +1,206 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Forms.UI.Controls
+{
+    /// <summary>
+    /// WinForms-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbar"/>
+    /// </summary>
+    [Designer(typeof(InkToolbarDesigner))]
+    public class InkToolbar : WindowsXamlHostBaseExt
+    {
+        protected Windows.UI.Xaml.Controls.InkToolbar UwpControl => XamlElement as Windows.UI.Xaml.Controls.InkToolbar;
```

Comment:
`private`

Reply:
Changing to Internal due to that Toolbar class accesses this as a member when setting its own member property TargetInkCanvas